test/e2e/networking.go

```diff
@@ -122,7 +78,54 @@ func LaunchNetTestPodPerNode(nodes *api.NodeList, name string, c *client.Client,
 }
 
 var _ = Describe("Networking", func() {
-	//var c *client.Client
+
+	var svcname = "nettest"
+	var c *client.Client = nil
+
+	BeforeEach(func() {
+		//Assert basic external connectivity.
+		//Since this is not really a test of kubernetes in any way, we
+		//leave it as a pre-test assertion, rather than a Ginko test.
+		By("Executing a successfull http request from the external internet")
+		resp, err := http.Get("http://google.com")
+		if err != nil {
+			Failf("Unable to connect/talk to the internet: %v", err)
+		}
+		if resp.StatusCode != http.StatusOK {
+			Failf("Unexpected error code, expected 200, got, %v (%v)", resp.StatusCode, resp)
+		}
+
+		By("Building a namespace api object")
+		var ns = svcname + "-" + randomSuffix()
+
+		namespaceObj = &api.Namespace{
+			ObjectMeta: api.ObjectMeta{
+				Name:      ns,
+				Namespace: "",
+			},
+			Status: api.NamespaceStatus{},
+		}
+
+		By("Creating a kubernetes client")
+		c, err = loadClient()
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Creating a namespace for this test suite")
+		_, err = c.Namespaces().Create(namespaceObj)
+		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
```

Comment:
seems to be no good way to reuse err w/o shadowing cli ... :( not sure what the right way to reuse error variable is.  decalring a new one.